requirements.txt

```diff
@@ -1,2 +1,3 @@
+Cython
```

Comment:
Do you understand what is using Cython?
Was this a requirement for whisper already?

Reply:
I'm pretty sure it was dtw-python. Whisper on it's own didn't require Cython. It was only when I was trying to use whisper-timestamped

Reply:
OK thanks a lot.
I merged your PR in master (partially by mistake...), and made the modifications I was suggesting on this PR